src/services/userService.js

```diff
@@ -0,0 +1,45 @@
+let users = [];
+
+const resetUsers = () => {
+  users = [];
+};
+
+const getAllUsers = () => {
+  return users;
+};
+
+const getUserById = (id) => {
+  return users.find((person) => person.id === +id) || null;
+};
+
+const createUser = (name) => {
+  const user = {
+    id: Math.floor(Math.random() * 1000),
```

Comment:
The way you're generating user IDs could lead to collisions. If two IDs happen to be the same, it will cause issues because IDs should be unique. Consider using a more reliable way to generate unique IDs, such as a library like uuid.